src/Fixer/Operator/BinaryOperatorSpacesFixer.php

```diff
@@ -29,66 +29,147 @@
  */
 final class BinaryOperatorSpacesFixer extends AbstractFixer implements ConfigurationDefinitionFixerInterface
 {
+    use AlignmentFixerHelperTrait;
```

Comment:
ah, finaly as a trait ;)
but actually... we have previous helper extracted as they were used in multiple fixers, now we manage to have one fixer to rule all cases.
would this trait really be used somewhere else ?
if not, there is no need to have it extracted (which encourage to use it somewhere else).

When we want to extract logic, we make separated **class**. it's good.
When we want to extract behavior (feature=trait), we extract it to **trait** - like "LoggerTrait" adds logging into any class it's injected. But that reusable logic.
Here, I'm not sure about alignmnet trait.